pytorch_lightning/accelerators/accelerator.py

```diff
@@ -33,7 +37,10 @@ class ReduceOp:
 
 class Accelerator(object):
 
-    def __init__(self, trainer=None, cluster_environment=None, ddp_plugin=None):
+    def __init__(self,
+                 trainer: Optional['pl.Trainer'] = None,
```

Comment:
```suggestion
                 trainer: Optional['Trainer'] = None,
```
 as it is string you do not need the pl

Reply:
You need it for the doc.

Reply:
Same it breaks the doc.